src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -350,7 +356,7 @@ void *frontendservicesesion(std::string masterIP, int connFd, SQLiteDBInterface
             string graphType(type);
             graphType = utils.trim_copy(graphType, " \f\n\r\t\v");
 
-            std::unordered_set<std::string> s = {"1", "2", "3"};
+            std::unordered_set <std::string> s = {"1", "2", "3"};
```

Comment:
What are these 1, 2, 3 values correspond to? Can we have some more meaningful list of items here?

Reply:
This section was there originally. the only change that has come from my PR is that adding a vertical space

Reply:
Ok